save-frontend/src/main/kotlin/com/saveourtool/save/frontend/App.kt

```diff
@@ -28,8 +28,10 @@ import com.saveourtool.save.frontend.utils.*
 import com.saveourtool.save.info.UserInfo
 import com.saveourtool.save.validation.FrontendRoutes
 
+import browser.document
```

Comment:
They seem to be moving to using same APIs from `kotlin-wrappers` instead of `kotlin-stdlib-js`, which sometimes makes types incompatible in compile time even though they are eventually mapped onto the same external declarations